submissions/akinbobolaFaith/database/init.sql

```diff
@@ -0,0 +1,142 @@
+("Skechers Men's Blaine Orsen Ankle Boot", 'Approx. 1'' heel height, Approx. 5'' boot shaft height, Cushioned footbed, Dual side goring Fabric lining', 'Skechers-Men-Blaine-Orsen-Ankle-Boot.jpg', 22000.00, 50, 'low stock', now()),
+("UGG Men's Biltmore Chelsea Boot", 'Imported, Rubber sole, Shaft measures approximately ankle-high from arch, Waterproof full-grain leather upper, Energ comfort system insole, White spider rubber outsole for added traction, Fully waterproof and seam-sealed construction, Textile lining', 'UGG-Men-Biltmore-Chelsea-Boot2.jpeg', 24999.00, 240, 'in stock', now()),
+('Fendi Custom Fotted Ladies Gown-Black', 'Casual, meet smart, Polo collar, Button placket, Embroidered logo to chest, Fitted cuffs, Slim fit, A narrow cut that sits close to the body', 'Fendi-Custom-Fotted-Ladies-Gown-Black.jpg', 20999.00, 0, 'out of stock', now());
+
+
+/*********************CUSTOMERS TABLE*************************/
+CREATE TABLE customers (
+	id INT NOT NULL AUTO_INCREMENT,
+	firstname VARCHAR(50) NOT NULL,
+	lastname VARCHAR(50) NOT NULL,
+	email_address VARCHAR(100) NOT NULL,
+	password VARCHAR(50) NOT NULL,
+	created_by DATETIME,
+	PRIMARY KEY (id)
+);
+
+INSERT INTO customers (firstname, lastname, email_address, password, created_by)
+VALUES ('Michael', 'Daniel', 'miichaeldaniel@gmail.com', 'Danmichael24.', now()),
+('Chukwu', 'Flora', 'chukwuflora@gmail.com', 'floRal58chuks.', now()),
+('Adeola', 'Abosede', 'bosededebola@gmail.com', 'Debolboss_1984', now()),
+('Benson', 'Bonaventure', 'bensonbonaventure@gmail.com', 'bona_Benson24', now()),
+('Bernard', 'Davidson', 'bernarddavid@gmail.com', 'bernardDave44.', now());
+
+
+/*********************CUSTOMERS ADDRESS TABLE*************************/
+CREATE TABLE customers_addresses (
+	id INT NOT NULL AUTO_INCREMENT,
+	customers_id INT NOT NULL,
+	street VARCHAR(100) NOT NULL,
+	city VARCHAR(50) NOT NULL,
+    state VARCHAR(50) NOT NULL,
+	zip_code INT,
+	country VARCHAR(50) NOT NULL,
+	phone_number VARCHAR(20) NOT NULL,
+    created_at DATETIME,
+	PRIMARY KEY (id),
+	FOREIGN KEY (customers_id) REFERENCES customers (id)
+);
+
+INSERT INTO customers_addresses (customers_id, street, city, state, zip_code, country, phone_number, created_at)
+VALUES (1, 'No 10, Cheraton Avenue', 'Ikeja', 'Lagos', 00145, 'Nigeria', '08012345600', now()),
+(2, 'No 1, Clinton Close', 'Ibadan', 'Oyo', 00130, 'Nigeria', '08012345601', now()),
+(3, 'No 3, Mobayode Street', 'Abeokuta', 'Ogun', 00120, 'Nigeria', '08012345602', now()),
+(4, 'No 7, Hamilton Avenue', 'Awka', 'Anambra', 00014, 'Nigeria', '08012345603', now()),
+(5, 'No 12, Paragon Close', 'Enugu', 'Enugu', 00173, 'Nigeria', '08012345604', now());
+ 
+ 
+/*********************CATEGORIES TABLE*************************/
+CREATE TABLE categories (
+    id INT NOT NULL AUTO_INCREMENT,
+	products_id INT NOT NULL,
```

Comment:
kindly read the instructions again thoroughly. you were told `each product belongs to a category` and not `each category belongs to a product`